### PR TITLE
Fix #56: Add a Contributor Dashboard info page to wiki for new contributors.

### LINKED ISSUE #56: Add instructions for populating contributor dashboard data on the local server
New contributors likely don't know how to submit video proof that their contributions don't break the contributor dashboard, because quite a bit of setup is needed. There are some instructions for populating the necessary data on a local server in [this doc](https://docs.google.com/document/d/1JYX4nvTcblaVVYAlTi7rApE0lWSBx0v_ZCCr_8WW4Wc/edit), but they're not very discoverable.

So, the aim of this issue is to add a wiki page explaining how to set up the data on the local server necessary for testing contributor-dashboard-related changes.

### PR DESCRIPTION
**Pull request by sagangwee**
We can then link to this wiki page in contributor dashboard starter issues. Fixes https://github.com/oppia/oppia-web-developer-docs/issues/56.

### CODE REVIEW

**Comment by sagangwee** [comment]
@seanlip PTAL, thanks!

File: Contributor-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## How items for contribution are populated
+
+### Translate text tab
+Every list item on the "Translate Text" tab corresponds to a particular lesson and all of its pieces of text content. In the codebase, we refer to each of these list items as _opportunities_. The contributor dashboard automatically shows a translation opportunity for a lesson when the following are true:
+
+1. The lesson (also called an _exploration_) corresponds to a _chapter_ in a _story_ of a published classroom subject, e.g. "Decimals" (also called a _topic_). See the [user docs](https://oppia-user-guide.readthedocs.io/en/latest/keyconcepts.html) for an overview of these terms.
+1. Every piece of text content in the lesson does not yet have an accepted translation.
```
Comment (seanlip) [inline Contributor-dashboard.md]: This sounds like the entire lesson is untranslated. Maybe instead: "There is at least one piece of text content in the lesson that does not yet have an accepted translation." ?

File: Contributor-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+1. allowlist a user to review question suggestions
+1. remove rights from a user for any of the above
+
+See [this](https://docs.google.com/document/d/1VqNiJttq85YyR6cQkd8M9lGGkOP8OlUlkI37Xw6SovM/edit) doc for step-by-step admin instructions. This may be useful for developing locally as a coder as well.
+
+## Local development
+Some setup is usually required when developing locally for the contributor dashboard since before a user can submit a content suggestion to a lesson, a lesson needs to exist. Additionally, the requirements outlined in [How items for contribution are populated](#how-items-for-contribution-are-populated) must be satisfied. See [this](https://docs.google.com/document/d/1JYX4nvTcblaVVYAlTi7rApE0lWSBx0v_ZCCr_8WW4Wc/edit#) doc for step-by-step instructions on how to populate test data when running a local server.
+
+## Appendix
+1. [Contributor dashboard overview](https://docs.google.com/document/d/1wM9cQzq1-3nbEhZliRlpnGDXbM_HspNkY16CYnA6lWg/edit#): More in-depth developer focused overview of the backend structure of the contributor dashboard.
```
Comment (seanlip) [inline Contributor-dashboard.md]: Just a heads-up that there's an unresolved comment in this doc.

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks @sagangwee! Looks fantastic, just two minor notes.

**Comment by seanlip** [comment]
@sagangwee I just had an idea -- what do you think about adding one or two simple exercises to this doc to help contributors verify that they've got things set up and can make changes? E.g. ask them to add a label to the right of each reviewer's name, or add a date to the card, or something simple like that, and show a screenshot of what they should see on their local machine.

This could help potential contributors get a feel for the ins and outs of the dashboard, and can also give you a check when accepting someone new to the team (you can ask for a screenshot of that toy project). Thoughts?

File: Contributor-dashboard.md (hunk 1 above)
Comment (sagangwee) [reply Contributor-dashboard.md]: Ah, yup. That reads better. Thanks. Done.

File: Contributor-dashboard.md (hunk 2 above)
Comment (sagangwee) [reply Contributor-dashboard.md]: Ack. I'll work on updating that.

**Comment by sagangwee** [comment]
@seanlip Ah, that reminds me—I forgot to add relevant code pointers with corresponding screenshots of the UI where applicable. I think once I add those, that should be enough for a new contributor to get their bearings.

**Review by sagangwee — COMMENTED** [review]
@seanlip Addressed your comments, thanks!

**Comment by seanlip** [comment]
> @seanlip Ah, that reminds me—I forgot to add relevant code pointers with corresponding screenshots of the UI where applicable. I think once I add those, that should be enough for a new contributor to get their bearings.

Ack, looks good, but how about adding an exercise as well? The reason I say this is because while the code pointers help folks find where to look for, they'd probably need to do something small to get a feel for the "flow" of data through the various components. 

So I would suggest adding a paragraph with the smallest "feature" you can think of that illustrates that and suggest that they optionally try working through it as a tutorial -- you can show them a screenshot of what they should see if they successfully get it running, and explain how to test their work. If people get stuck with that, then they'll post on GitHub Discussions and it'll help us get a sense of where the onboarding blockers are. Wdyt?

(Otherwise, everything here looks great, thanks!)

**Comment by seanlip** [comment]
@sagangwee Can we get this in? It might be useful for #16740 on the main repo.

**Comment by sagangwee** [comment]
@seanlip Sorry, was not able to get to this today. Will address tomorrow!

**Review by sagangwee — COMMENTED** [review]
@seanlip Added a sample coding exercise. PTAL, thanks!

**Review by seanlip — APPROVED** [review]
It looks great, thanks @sagangwee!

Could you please remove the .DS_Store that seems to have been included accidentally? Then let's get this merged :) Thanks!

**Comment by seanlip** [comment]
Merged. Thanks @sagangwee!

**Pull request merged**